docs/running.rst

```diff
@@ -22,6 +22,22 @@ To run on 4 machines with 4 GPUs each:
 
     $ horovodrun -np 16 -H server1:4,server2:4,server3:4,server4:4 python train.py
 
+Host nodes can also be specified in a host file:
+For example, 
+
+$ cat myhostfile
+aa slots=2
+bb slots=2
+cc slots=2
+
+Here, we list both the host names (aa, bb, and cc) but also how many "slots" there are for each.
+Slots indicate how many processes can potentially execute on a node.
+This format is the same as in mpirun command,
```

Comment:
as --> as the format used in the 

Recommend making "the mpirun command" the link text